chainstate/test-framework/src/utils.rs

```diff
@@ -264,3 +265,42 @@ fn new_token_burn_output(
         OutputPurpose::Burn,
     )
 }
+
+// TODO: Replace by a proper UTXO set abstraction
+// (https://github.com/mintlayer/mintlayer-core/issues/312).
+pub type TestBlockUtxos = Vec<(OutPointSourceId, Vec<TxOutput>)>;
+
+fn utxos_from_genesis(genesis: &Genesis) -> TestBlockUtxos {
+    vec![(
+        OutPointSourceId::BlockReward(genesis.get_id().into()),
+        genesis.utxos().to_vec(),
+    )]
+}
+
+fn utxos_from_block(blk: &Block) -> TestBlockUtxos {
+    let utxos = std::iter::once((
+        OutPointSourceId::BlockReward(blk.get_id().into()),
+        blk.block_reward().outputs().to_vec(),
+    ))
+    .chain(blk.transactions().iter().map(|tx| {
+        (
+            OutPointSourceId::Transaction(tx.transaction().get_id()),
+            tx.transaction().outputs().clone(),
+        )
+    }))
+    .collect();
+    utxos
+}
+
+impl TestFramework {
```

Comment:
Please put this in framework.rs, with the other impls. It's unidiomatic to separate impls in different modules.

Reply:
You are right, fixed, thanks.